services/order.py

```diff
@@ -0,0 +1,34 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> None:
+    with transaction.atomic():
+        user = get_user_model().objects.get(username=username)
+        order = Order.objects.create(user=user)
+        if date:
+            order.created_at = datetime.strptime(date, "%Y-%m-%d %H:%M")
+            order.save()
+        for ticket in tickets:
+            Ticket.objects.create(
+                movie_session_id=ticket["movie_session"],
+                order=order,
+                row=ticket["row"],
+                seat=ticket["seat"]
+            )
```

Comment:
use bulk_create()
https://stackoverflow.com/questions/3608487/bulk-create-model-objects-in-django

Reply:
agreed in slack chat with @ArsenPidhoretskyi to leave with context manager